schemas/commands/lpa-questionnaire.schema.json

```diff
@@ -11,157 +11,20 @@
-              },
-              "neighbouringSiteAddressLine2": {
-                "type": ["string", "null"],
-                "description": "Second line of address of the site"
-              },
-              "neighbouringSiteAddressTown": {
-                "type": "string",
-                "examples": ["Maidstone"],
-                "description": "Town / City of the site address"
-              },
-              "neighbouringSiteAddressCounty": {
-                "type": ["string", "null"],
-                "examples": ["Kent"],
-                "description": "County of the site address"
-              },
-              "neighbouringSiteAddressPostcode": {
-                "type": "string",
-                "examples": ["MD21 5XY"],
-                "description": "Postal code of the site address"
-              },
-              "neighbouringSiteAccessDetails": {
-                "type": ["string", "null"],
-                "examples": ["Ongoing road works at this address"],
-                "description": "Provided information on site accessibility on this address"
-              },
-              "neighbouringSiteSafetyDetails": {
-                "type": ["string", "null"],
-                "examples": ["Ongoing road works at this address"],
-                "description": "Provided information on site health and safety on this address"
-              }
-            },
-            "required": [
-              "neighbouringSiteAddressLine1",
-              "neighbouringSiteAddressLine2",
-              "neighbouringSiteAddressTown",
-              "neighbouringSiteAddressCounty",
-              "neighbouringSiteAddressPostcode",
-              "neighbouringSiteAccessDetails",
-              "neighbouringSiteSafetyDetails"
-            ]
-          },
-          "description": "A list of neighbouring site addresses"
-        },
-        "affectedListedBuildingNumbers": {
-          "type": ["array", "null"],
-          "items": {
-            "type": "string",
-            "examples": ["10023", "17824"]
-          },
-          "description": "A list of affected listed building IDs from Historic England"
-        },
-        "lpaCostsAppliedFor": {
-          "type": ["boolean", "null"],
-          "description": "Indicates if the LPA has applied for costs"
-        }
-      },
-      "required": [
-        "caseReference",
-        "lpaQuestionnaireSubmittedDate",
-        "lpaStatement",
-        "siteAccessDetails",
-        "siteSafetyDetails",
-        "isCorrectAppealType",
-        "isGreenBelt",
-        "inConservationArea",
-        "notificationMethod",
-        "newConditionDetails",
-        "nearbyCaseReferences",
-        "neighbouringSiteAddresses",
-        "affectedListedBuildingNumbers",
-        "lpaCostsAppliedFor"
+      "anyOf": [
+        {"allOf": [
+          { "$comment": "backwards compatibility for no case type, when removed can update allOf to oneOf" },
```

Comment:
any of, needed to allow backwards compatibility
used not required to ensure the additional properties doesn't lead to S78 cases being validated by the HAS model